clubs_guides/urls.py

```diff
@@ -1,3 +1,18 @@
 from django.conf.urls import url
 
-# Create you url mappings here.
+from clubs_guides.views import (
+    ClubsGuidesListView,
+    ClubsGuideView,
+    CategoryListView,
+)
+
+
+urlpatterns = [
```

Comment:
remember that not a lot of people at mofo actually *know* django at this point, so overdocumenting and/or (probably "and" =) explicitly defining routes rather than regexp is low effort, high value code.